backend/fms_core/resources/sample.py

```diff
@@ -59,6 +59,8 @@ class SampleResource(GenericResource):
 
     volume = Field(attribute='volume', column_name='Volume (uL)', widget=DecimalWidget())
 
+    project = Field(attribute='projects', column_name='Project', widget=ManyToManyWidget(Project))
```

Comment:
If you are manually linking in the project, do not set the attribute and widjet. It will not help you since those are there to automatically create many-to-many entries when possible (not the case here). You may have to manually add project to the confirmation columns (by adding it in the export order).